kAI-api/src/middleware/error-handler.ts

```
@@ -0,0 +1,114 @@
+import { Request, Response, NextFunction } from 'express';
+import { ApiError } from '../errors/api-error';
+
+/**
+ * Global error handler middleware for Express
+ */
+export function errorHandler(
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void | Response {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)